### DISCUSSION TITLE
请问如何将参考文献的题目改成首字母大写？

### DISCUSSION THREAD

**Discussion by AgnesLee97** [tuna/thuthesis#765] [Q&A] [answered]
如题，参考文献论文题目如何改成首字母大写？

**Comment by AgnesLee97** [comment, marked as answer]
我找到了一个方法！可以批量修改，而且不会把连续的大写字母（即专有名词的缩写）改成小写
https://github.com/redleafnew/zotero-javascripts

**Comment by zepinglee** [comment]
你说的「首字母大写」是指 [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case) 还是 [title case](https://en.wikipedia.org/wiki/Letter_case#Title_case)？

《写作指南》和 GB/T 7714—2015 没有直接指出使用 sentence case 还是 title case。但是从给出的示例来看，标题使用 sentence case，但析出文献的书名（如会议论文集名）使用 title case。

**Reply by AgnesLee97** [reply]
我说的是把标题都改成sentence case，因为从各个网站导出bibtex再导入zotero之后，有的论文标题是sentence case，有的是titile case，用上面那个方法可以把它们统一起来

**Reply by zepinglee** [reply]
Zoetero 的标准用法是在数据库中使用 sentence case 著录（<https://www.zotero.org/support/kb/sentence_casing>），但是在 `.bib` 数据库中应当使用 title case，并且用大括号将专有名词保护起来（<https://tex.stackexchange.com/a/140071/82731>）。Zotero 的 BetterBibTeX 插件在导出 `.bib` 可以自动完成这个转换。

**Reply by AgnesLee97** [reply]
哦哦，感谢！我试试
